frontend/src/scenes/session-recordings/player/playerSettingsLogic.ts

```diff
@@ -185,6 +185,7 @@ export const playerSettingsLogic = kea<playerSettingsLogicType>([
         setShowFilters: (showFilters: boolean) => ({ showFilters }),
         setPrefersAdvancedFilters: (prefersAdvancedFilters: boolean) => ({ prefersAdvancedFilters }),
         setQuickFilterProperties: (properties: string[]) => ({ properties }),
+        setShowRecordingListPreviews: (enabled: boolean) => ({ enabled }),
```

Comment:
I think I'd still call it metadata or properties or something

preview feels more like I'm going to see the recording than info about it